jgiven-gradle-plugin/src/test/groovy/com/tngtech.jgiven.gradle/JGivenPluginShould.groovy

```diff
@@ -18,6 +18,37 @@ class JGivenPluginShould extends Specification {
         buildFile = new File(testProjectDir,'build.gradle')
     }
 
+    def "configuration is cacheable"() {
+        given:
+        buildFile << """
+        plugins {
+            id 'java'
+            id 'com.tngtech.jgiven.gradle-plugin'
+        }
+        
+       repositories { mavenCentral() }
+       dependencies {
+         testImplementation 'com.tngtech.jgiven:jgiven-junit:1.2.5' 
+         testImplementation 'junit:junit:4.13.2' 
+        }
+    """
+
+        new File(testProjectDir,"src/test/java").mkdirs()
+        File scenario = new File(testProjectDir,"src/test/java/SimpleScenario.java")
+        scenario << Resources.toByteArray(Resources.getResource("SimpleScenario.java"))
+
+        when:
+        def result = GradleRunner.create()
+                .withProjectDir(testProjectDir)
+                .withArguments("--configuration-cache", "test", "jgivenTestReport", "-S", "--info"/*, "-Dorg.gradle.debug=true"*/ )
```

Comment:
Will this comment still have some purpose?

Reply:
Damn, I totally missed that.  but @jjohannes  thanks for the suggestion